rules/sources/contact_data.yaml

```diff
@@ -25,7 +25,7 @@ sources:
     isSensitive: False
     sensitivity: medium
     patterns:
-      - "(?i).*((phone|mobile|cellphone|contact)[^\\s/(;)#|,=!>]{0,2}(detail|details)|(?:mobile|phone|cellphone|contact)[^\\s/(;)#|,=!>]{0,10}(?:number|nbr|num|no)|(?:primary|secondary)[^\\s/(;)#|,=!>]{0,10}(phone|landline|telephone))"
+      - "(?i)(((^(?!\s*$))(phone|mobile|cellphone|contact))|.*(phone|mobile|cellphone|contact)[^\\s/(;)#|,=!>]{0,2}(?:detail|details)|(?:mobile|phone|cellphone|contact)[^\\s/(;)#|,=!>]{0,10}(?:number|nbr|num|no)|(?:primary|secondary)[^\\s/(;)#|,=!>]{0,10}(phone|landline|telephone))"
```

Comment:
1. Adding a `(?!$)` will not match with variables named `phone`
2. The updated regex does not match with `employee_phone_number` any more. This seems the result of parentheses being removed from around the entire older regex. The `.*` is now a part of the first alternative `.*(phone|mobile|cellphone|contact)[^\\s/(;)#|,=!>]{0,2}(?:detail|details)`, while it should come before all the alternatives followed something like : `.*((phone|mobile|cellphone|contact)[^\\s/(;)#|,=!>]{0,2}(?:detail|details)|(?:mobile|phone|cellphone|contact)[^\\s/(;)#|,=!>]{0,10}(?:number|nbr|num|no)|(?:primary|secondary)[^\\s/(;)#|,=!>]{0,10}(phone|landline|telephone))`